Makefile

```diff
@@ -140,9 +139,10 @@ install-schemas:
 	install -Dm644 "$(DEV_DATA_DIR)"/*.glade "$(DESTDIR)$(GLADE_DIR)/"
 	install -dm755                                         "$(DESTDIR)$(SCHEMA_DIR)"
 	install -Dm644 "$(DEV_DATA_DIR)/org.guake.gschema.xml" "$(DESTDIR)$(SCHEMA_DIR)/"
+	echo "$$(python setup.py --version)" > "$(DESTDIR)$(SCHEMA_DIR)/version"
```

Comment:
What is the difference between this command and `try_to_compile_glib_schemas()`? It seems they perform same thing?

Reply:
Same question here, will `python setup --version`  generate [.dev(num)]?

Reply:
When we `sudo make install`, the schema files are installed to a place that needs root privileges to modify, which is unchangeable if you don't run guake with `sudo guake`. Notice how the `compile-schemas` step in `sudo make install` is also a duplicate of functionality that was already in try_to_compile_glib_schemas(), for the same reason.

Reply:
Also yeah, setup.py --version makes the version number that we see in guake_version()

Reply:
got it